packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -0,0 +1,623 @@
+import { BigNumber } from 'bignumber.js'
+import * as React from 'react'
+import { StyleSheet, Text as RNText, View } from 'react-native'
+import CopyToClipboard from 'src/dev/CopyToClipboard'
+import ProgressCutBar from 'src/dev/ProgressCutBar'
+import { H1 } from 'src/fonts/Fonts'
+import { I18nProps, withNamespaces } from 'src/i18n'
+import Chevron, { Direction } from 'src/icons/chevron'
+import { HEADER_HEIGHT } from 'src/shared/Styles'
+import { colors, standardStyles, textStyles, typeFaces } from 'src/styles'
+import { cutAddress, formatNumber, weiToDecimal } from 'src/utils/utils'
+
+class Text extends RNText {
+  render() {
+    return <RNText style={[styles.defaultText, this.props.style]}>{this.props.children}</RNText>
+  }
+}
+
+interface HeaderCellProps {
+  style: any[]
+  name: string
+  order: boolean | null
+  onClick: () => void
+}
+
+const HeaderCell = React.memo(function HeaderCellFn({
+  style,
+  name,
+  order,
+  onClick,
+}: HeaderCellProps) {
+  return (
+    <View onClick={onClick} style={[styles.tableHeaderCell, ...((style || []) as any)]}>
```

Comment:
can you change this to `onPress` for the prop name. (variable im not so concerned with)

Reply:
its just more idiomatic for React native web. 

and i think RNW even doesn't guarantee it wont break onClick (but i might be wrong)

Reply:
`onPress` shows a TypeScript error used on `View` and `onClick` is working fine. What could produce that?

Reply:
hmm

oh right I forgot `onPress` only applies to `Text` and `Touchable*`s  

nevermind